server/middlewares/controllers/authController.js

```diff
@@ -1,16 +1,32 @@
 const User = require("../../models/User");
+const jwt = require("jsonwebtoken");
 
 const signUp = async (req, res) => {
   const { firstName, lastName, password, email } = req.body;
 
-  await User.query().insert({
+  const user = await User.query().insert({
     firstName,
     lastName,
     password,
     email
   });
 
-  res.status(201).jsend({ message: "User registered" });
+  if (user) {
```

Comment:
Let's remove this check. there's always going to be a user.

Reply:
ok